forge-gui/res/cardsfolder/upcoming/the_master_mesmerist.txt

```diff
@@ -0,0 +1,11 @@
+Name:The Master, Mesmerist
+ManaCost:2 U B
+Types:Legendary Creature Time Lord Rogue
+PT:3/3
+A:AB$ Pump | Cost$ T | ValidTgts$ Creature.OppCtrl+powerLEX | TgtPrompt$ Select target creature that opponent controls | KW$ Skulk | SubAbility$ DBGoad | StackDescription$ {c:Targeted} gains skulk until end of turn. Goad it. | SpellDescription$ Target creature an opponent controls with power less than or equal to NICKNAME's power gains skulk until end of turn. Goad it. (A creature with skulk can't be blocked by creatures with greater power.)

```

Comment:
TgtPrompt should probably say something about the power thing

Cards that reference themselves in defining valid targets are becoming more common – probably need to add CARDNAME and NICKNAME substitutions to ValidTgts @tool4ever ?

Reply:
I was under the impression at least the former was always being handled

Reply:
Ah I see Earthshaker Khenra. @Glorax for now you should template the TgtPrompt like that card (no `.` though) but I think you'll have to use NICKNAME instead of cardname